Exploration Queries/InputEntity_Account/UserAccount_SubsOwnedDirectly.yaml

```diff
@@ -0,0 +1,24 @@
+Id: d103d814-eb0f-4395-af45-7f5697e598aa
+DisplayName: "Subscriptions owned by the user"
+Description: "Returns all subscriptions owned directly by the user"
+InputEntityType: Account 
+InputFields:
+  - AadUserId 
+OutputEntityTypes:
+  - AzureResource  
+QueryPeriodBefore: 7d 

```

Comment:
Normally we want only 12h before and 12h after, 7d lookback in an expansion query but cause the graph to hang